### PR TITLE
Website layout

### PR DESCRIPTION
**Pull request by Mvalverde00**
One of the commits here undoes a fix to one of the suggestions made in [Pull Request 3](https://github.com/googleinterns/step176-2020/pull/3#discussion_r459012168).  This is because it causes the server to crash with the following error:
`java.lang.IllegalAccessException: class org.eclipse.jetty.server.handler.ContextHandler$StaticContext cannot access a member of class com.google.sps.servlets.AggregationServlet with modifiers ""`
Reinserting the `public` keyword solved the issue, but I am still unsure what was causing it in the first place and why the error does not occur in the tests.  Perhaps it has something to do with how the framework creates the servlets?

### CODE REVIEW

**Review by georgezheng999 — APPROVED** [review]

File: src/main/webapp/dashboard-manager.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
+class DashboardManager {
+  constructor() {
+    this.data = this.initData();
+
+    this.dashboard = new google.visualization.Dashboard(document.getElementById('dashboard'));
+    this.aggregationSelector = createNewAggregationSelector();
+    this.table = createNewTable()
```
Comment (ChuxiangP) [inline src/main/webapp/dashboard-manager.js]: missing ```;```

File: src/main/webapp/dashboard-manager.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                this.data.addRow([key, val]);
+              }
+      }));
+    } else {
+      // Setup data for standard table view
+      this.data.addColumn('string', 'Serial Number');
+      this.data.addColumn('string', 'Status');
+      this.data.addColumn('string', 'Asset ID');
+      this.data.addColumn('string', 'User');
+      this.data.addColumn('string', 'Location');
```
Comment (ChuxiangP) [inline src/main/webapp/dashboard-manager.js]: Looks like these and the addRow below can be replaced by initData().

File: src/main/webapp/dashboard-manager.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+      'view': {'columns': [0, 1]}
+  });
+}
+
+function createNewAggregationSelector() {
+  const aggregationOptions = new google.visualization.DataTable();
+  aggregationOptions.addColumn('string', 'Aggregation Options');
+  aggregationOptions.addRows([['annotatedAssetId'], ['annotatedLocation'], ['annotatedUser']]);
+
+  return new google.visualization.ControlWrapper({
```
Comment (ChuxiangP) [inline src/main/webapp/dashboard-manager.js:129]: Seems like we only support aggregate by one field at the moment. But I saw we can add multiple fields in UI during today's demo. Is there anything we can do to disable adding multiple fields until we support the functionality? 

If it's a big change I'm okay with leaving a todo for now.

Comment (Mvalverde00) [reply src/main/webapp/dashboard-manager.js:129]: Thankfully it was a simple change, the Google Charts API CategoryFilter class has an option called ["ui.allowMultiple"](https://developers.google.com/chart/interactive/docs/gallery/controls#options) that I could set to false.

**Review by ChuxiangP — APPROVED** [review]

File: src/main/webapp/dashboard-manager.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+  /* Get the initial data to be shown to the user and populate the main datatable*/
+  initData() {
+    let data = new google.visualization.DataTable();
+    data.addColumn('string', 'Serial Number');
+    data.addColumn('string', 'Status');
+    data.addColumn('string', 'Asset ID');
+    data.addColumn('string', 'User');
+    data.addColumn('string', 'Location');
+
+    data.addRow(['SN12345', 'Provisioned', '1e76c3', 'James', 'Texas']);
```
Comment (qlyao) [inline src/main/webapp/dashboard-manager.js:23]: Please comment that these are test data and consider adding a TODO here that replaces the test data with real data.

**Review by qlyao — APPROVED** [review]

File: src/main/webapp/dashboard-manager.js (hunk 4 above)
Comment (georgezheng999) [reply src/main/webapp/dashboard-manager.js:23]: added

**Pull request merged**